libs/blocks/merch/merch.js

```diff
@@ -671,7 +671,7 @@ export async function getCheckoutAction(offers, options, imsSignedInPromise, el)
 
 export function setPreview(attributes) {
   const { host } = window.location;
-  if (host.includes(`${SLD}.page`) || host.origin === 'https://www.stage.adobe.com') {
+  if (host.includes(`${SLD}.page`) || host === 'www.stage.adobe.com') {
```

Comment:
Is this used on business.adobe.com as well? Might as well cut the www. part

Reply:
bacom doesn't use merch prices afaik.
however i still like the idea to cut the www. part, it shouldn't hurt and we would be future proof in case bacom will start using our pricing

Reply:
Ye I agree with CodeQL as well (& in fact, already assumed codeQL would complain), though you could use something like `endsWith(.stage.adobe.com)` 
See https://codeql.github.com/codeql-query-help/python/py-incomplete-url-substring-sanitization/